<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KDRe</class>
 <widget class="QMainWindow" name="KDRe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KDRe - Bulk Image Resizer</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>390</y>
      <width>55</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Width</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>390</y>
      <width>55</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Height</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="widthEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>390</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="heightEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>390</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>331</width>
      <height>161</height>
     </rect>
    </property>
    <widget class="QLabel" name="sliderLabel">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>55</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>100%</string>
     </property>
     <property name="buddy">
      <cstring>sizeSlider</cstring>
     </property>
    </widget>
    <widget class="QRadioButton" name="pixelRadio">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>95</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="percentageRadio">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>95</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Percentage</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSlider" name="sizeSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Resize by</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Aspect Ratio</string>
     </property>
    </widget>
    <widget class="QComboBox" name="aspectRatio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>301</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="startResize">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>470</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QToolButton" name="resetList">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>470</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>671</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>331</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>File Info</string>
    </property>
    <widget class="QLineEdit" name="outputFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="browseOut">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>78</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Output Directory</string>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="fileList">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>180</y>
      <width>311</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="removeItem">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>390</y>
      <width>30</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>160</y>
      <width>55</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>File List</string>
    </property>
   </widget>
   <widget class="QLabel" name="titleIcon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>661</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>KDRe - Batch image resizer</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputFile">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browseIn">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>100</y>
      <width>78</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="inDirLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Add file or directory</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="useDir">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Use directory</string>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>groupBox</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>widthEdit</zorder>
   <zorder>heightEdit</zorder>
   <zorder>startResize</zorder>
   <zorder>resetList</zorder>
   <zorder>progressBar</zorder>
   <zorder>fileList</zorder>
   <zorder>removeItem</zorder>
   <zorder>label</zorder>
   <zorder>titleIcon</zorder>
   <zorder>inputFile</zorder>
   <zorder>browseIn</zorder>
   <zorder>inDirLabel</zorder>
   <zorder>useDir</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>695</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
